showcase-docs/corporate-proxy.md

```diff
@@ -0,0 +1,100 @@
+# Running the Showcase application behind a corporate proxy
+
+Out of the box, the Showcase application can be run behind a corporate proxy, by setting any of the following environment variables prior to starting the application:
+
+- `HTTP_PROXY`: HTTP proxy to use.
+- `HTTPS_PROXY`: distinct proxy to use for HTTPS requests.
+
+Additionally, you can set the `NO_PROXY` environment variable to exclude certain domains from proxying. The value is a comma-separated list of hostnames that do not require a proxy to get reached, even if one is specified.
+
+## Helm deployment
+
+You can set the proxy information in your Helm `values` file, like so:
+
+```yaml
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: HTTP_PROXY
+        # HTTP proxy to use
+        value: '<my_http_proxy_url>'
+      - name: HTTPS_PROXY
+        # Distinct proxy to use for HTTPS requests
+        value: '<my_https_proxy_url>'
+      - name: NO_PROXY
+        # Pattern of comma-separated URLs that should be excluded from proxying.
+        # Example: 'foo.com,baz.com'
+        value: '<my_no_proxy_settings>'
+```
+
+<details>
+<summary>Example</summary>
+
+```yaml
+# --- Truncated ---
+upstream:
+  backstage:
+    extraEnvVars:
+      - name: HTTP_PROXY
+        value: 'http://10.10.10.105:3128'
+      - name: HTTPS_PROXY
+        value: 'http://10.10.10.106:3128'
+      - name: NO_PROXY
+        value: 'localhost,example.org'
+```
+
+</details>
+
+## Operator deployment
```

Comment:
I think it worth mention that it can be configured as default config on cluster level.
It looks like a better place, indeed, it looks like infrastructural specific and would be applicable for the whole cluster,.

Reply:
Done in [`27b84b9` (#1225)](https://github.com/janus-idp/backstage-showcase/pull/1225/commits/27b84b9849a72fdcafb45a977ad659f0a926c8a8)